src/main/kotlin/com/wgtwo/example/voicemail/VoicemailDemo.kt

```diff
@@ -40,11 +40,10 @@ object VoicemailDemo {
         println(tempFile.absoluteFile)
         val outputStream = tempFile.outputStream()
 
-        if (voicemail.bytesCase != BytesCase.WAV) {
-            println("Unexpected format for voicemail.")
-            return
+        when (voicemail.bytesCase) {
+            BytesCase.WAV -> voicemail.wav.writeTo(outputStream)
+            BytesCase.BYTES_NOT_SET, null -> println("No content found in the voicemail response.")
```

Comment:
What happens when you later try to play a file with nothing in it?

Reply:
Good catch, that failed miserably. I updated the code now.